src/Apollo/ReactionsValidator.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Apollo;
+
+use Gdbots\Pbj\Assertion;
+use Gdbots\Pbj\MessageResolver;
+use Gdbots\Pbj\WellKnown\NodeRef;
+use Gdbots\Pbjx\DependencyInjection\PbjxValidator;
+use Gdbots\Pbjx\Event\PbjxEvent;
+use Gdbots\Pbjx\EventSubscriber;
+use Gdbots\Schemas\Ncr\Enum\NodeStatus;
+
+class ReactionsValidator implements EventSubscriber, PbjxValidator
+{
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            'triniti:apollo:command:add-reactions.validate' => 'validateAddReactions',
+            'triniti:apollo:mixin:reactions.validate'       => 'validateReactions',
+        ];
+    }
+
+    public function validateReactions(PbjxEvent $pbjxEvent): void
```

Comment:
@bhsiao55 not sure why this is needed.  also note, the has should also determine empty so two checks should not be needed.  but is this method needed at all?

Reply:
@gdbrown I was using this method to add the initial reaction types to the reactions node and set the count of each type to 0 when it is created

Reply:
@bhsiao55 is that necessary?

Reply:
@gdbrown what happens when I try to update a reaction type but that reaction type does not exist in the reactions node?

Reply:
@bhsiao55 pretty sure dynamodb uses zero as the initial value.  ref https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html.  but test it out.